### PR TITLE
VR > List View & Day View > Address Meta

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/131086
🎥 https://www.loom.com/share/4c41026cdff348efa2f8fc5cb7adb326

### CODE REVIEW

File: src/views/v2/day/event/venue.php [hunk 1]
```diff
@@ -9,25 +9,29 @@
...
 }
+
+$separator            = esc_html_x( ', ', 'Address separator', 'events-calendar-pro' );
+$venue                = $event->venues[0];
+$append_after_address = array_filter( array_map( 'trim', [ $venue->city, $venue->state_province, $venue->state, $venue->province ] ) );
 ?>
 <address class="tribe-events-calendar-day__event-venue tribe-common-b2">
 	<span class="tribe-events-calendar-day__event-venue-title tribe-common-b2--bold">
-		<?php echo isset( $venue_details['linked_name'] ) ? $venue_details['linked_name'] : esc_html__( 'Venue Name', 'the-events-calendar' ); ?>
+		<?php echo esc_html( $venue->post_title ) ?>
```
Comment (tr1b0t) [inline src/views/v2/day/event/venue.php]: :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 4f5615338d4d81f05ce866c559633a23</sub>

File: src/views/v2/day/event/venue.php [hunk 2]
```diff
@@ -9,25 +9,29 @@
...
 	</span>
 	<span class="tribe-events-calendar-day__event-venue-address">
-		<?php echo isset( $venue_details['address'] ) ? $venue_details['address'] : ''; ?>
+		<?php echo esc_html( $venue->address . ( $venue->address && $append_after_address ? $separator : '' ) ); ?>
+		<?php if ( $append_after_address ) : ?>
+			<?php echo esc_html( reset( $append_after_address ) ) ?>
```
Comment (tr1b0t) [inline src/views/v2/day/event/venue.php]: :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 28697c859ed9d117dce1388c26ca6a11</sub>

File: src/views/v2/list/event/venue.php [hunk 3]
```diff
@@ -9,25 +9,29 @@
...
 }
+
+$separator            = esc_html_x( ', ', 'Address separator', 'events-calendar-pro' );
+$venue                = $event->venues[0];
+$append_after_address = array_filter( array_map( 'trim', [ $venue->city, $venue->state_province, $venue->state, $venue->province ] ) );
 ?>
 <address class="tribe-events-calendar-list__event-venue tribe-common-b2">
 	<span class="tribe-events-calendar-list__event-venue-title tribe-common-b2--bold">
-		<?php echo isset( $venue_details['linked_name'] ) ? $venue_details['linked_name'] : esc_html__( 'Venue Name', 'the-events-calendar' ); ?>
+		<?php echo esc_html( $venue->post_title ) ?>
```
Comment (tr1b0t) [inline src/views/v2/list/event/venue.php]: :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: c4a38bb8da0b89b1f1cb033eb2dd08a0</sub>

File: src/views/v2/list/event/venue.php [hunk 4]
```diff
@@ -9,25 +9,29 @@
...
 	</span>
 	<span class="tribe-events-calendar-list__event-venue-address">
-		<?php echo isset( $venue_details['address'] ) ? $venue_details['address'] : ''; ?>
+		<?php echo esc_html( $venue->address . ( $venue->address && $append_after_address ? $separator : '' ) ); ?>
+		<?php if ( $append_after_address ) : ?>
+			<?php echo esc_html( reset( $append_after_address ) ) ?>
```
Comment (tr1b0t) [inline src/views/v2/list/event/venue.php]: :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: b43c33680102ef0fe06b885c2b9eff22</sub>

File: src/views/v2/day/event/venue.php [hunk 5]
```diff
@@ -9,25 +9,30 @@
...
+
+$separator            = esc_html_x( ', ', 'Address separator', 'events-calendar-pro' );
+$venue                = $event->venues[0];
+$append_after_address = array_filter( array_map( 'trim', [ $venue->city, $venue->state_province, $venue->state, $venue->province ] ) );
+$address              = $venue->address . ( $venue->address && $append_after_address ? $separator : '' );
 ?>
 <address class="tribe-events-calendar-day__event-venue tribe-common-b2">
 	<span class="tribe-events-calendar-day__event-venue-title tribe-common-b2--bold">
-		<?php echo isset( $venue_details['linked_name'] ) ? $venue_details['linked_name'] : esc_html__( 'Venue Name', 'the-events-calendar' ); ?>
+		<?php echo esc_html( $venue->post_title ); ?>
```
Comment (bordoni) [inline src/views/v2/day/event/venue.php]: We cant have `esc_html` on Post title. We allow HTML on titles.

File: src/views/v2/list/event/venue.php [hunk 6]
```diff
@@ -9,25 +9,30 @@
...
+
+$separator            = esc_html_x( ', ', 'Address separator', 'events-calendar-pro' );
+$venue                = $event->venues[0];
+$append_after_address = array_filter( array_map( 'trim', [ $venue->city, $venue->state_province, $venue->state, $venue->province ] ) );
+$address              = $venue->address . ( $venue->address && $append_after_address ? $separator : '' );
 ?>
 <address class="tribe-events-calendar-list__event-venue tribe-common-b2">
 	<span class="tribe-events-calendar-list__event-venue-title tribe-common-b2--bold">
-		<?php echo isset( $venue_details['linked_name'] ) ? $venue_details['linked_name'] : esc_html__( 'Venue Name', 'the-events-calendar' ); ?>
+		<?php echo esc_html( $venue->post_title ); ?>
```
Comment (bordoni) [inline src/views/v2/list/event/venue.php]: We cant have `esc_html` on Post title. We allow HTML on titles.

**Pull request merged**